.mailmap

```diff
@@ -242,29 +246,34 @@ Günther Jungbluth <gunther@gameslabs.net>
 Hakan Özler <hakan.ozler@kodcu.com>
 Hao Shu Wei <haosw@cn.ibm.com>
 Hao Shu Wei <haosw@cn.ibm.com> <haoshuwei1989@163.com>
+Hao Shu Wei <haosw@cn.ibm.com> <haoshuwei24@gmail.com>
```

Comment:
Of these three, it looks like the `gmail.com` address is the most current (looking at commit history for all three of them); it looks like this contributor is now at Alibaba, so perhaps we should make the gmail address the canonical one; also looks like that's the address mentioned in their profile; https://github.com/haoshuwei

Reply:
Good catch, I got thrown off on this one due to the duplicate entry in the mailmap later that I didn't catch at first :laughing: